app/controllers/users_controller.rb

```diff
@@ -1,4 +1,5 @@
-class UsersController < ActionController::Base
+class UsersController < ApplicationController
+  before_action :authenticate_user!, except: [:index]
   def index
     @users = User.all.order(id: :asc)
   end
```

Comment:
- You have done well so far, but I can still access the `user#index` page after signing out of your application. This is because you have made an authentication exception for the index controller. Please make sure a user is authenticated before accessing any controller action. Please fix this 👍 👍

Reply:
ok thank you